### DISCUSSION TITLE
Auto-focus on search box interferes with keyboad navigation

### DISCUSSION THREAD

**Discussion by timmc** [rust-lang/crates.io#9701] [Feature Requests]
When I land on a crates.io page like https://crates.io/crates/anyhow one of the very first things I want to do is start scrolling down the page with space bar, page down, or down arrow. But I can't, since the search field is already focused and is capturing those keystrokes -- so I have to tab out, or even click somewhere if I don't want the space bar to start activating random links.

Would you be open to disabling this auto-focus? Perhaps with the exception of the main page.

**Comment by Turbo87** [comment]
> Would you be open to disabling this auto-focus? Perhaps with the exception of the main page.

I think I'd be fine with disabling it on child pages
